exasol_transformers_extension/utils/dataframe_operations.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+from typing import List,  TypeVar
+
+T = TypeVar('T')
+
+
+def get_sorted_unique_values(
+        df: pd.DataFrame, columns: List[T]) -> List[List[T]]:
+    """
+    Filter given dataframe and return the sorted unique values
+
+    :param df: Dataframe from which the uniques values are extracted
+    :param columns: List of columns to be filtered and sorted accordingly
+    """
+
+    return df[columns].drop_duplicates().sort_values(by=columns).values.tolist()
```

Comment:
I think, in the order by and here we should explicit define if we are ascending or descending sorting, because they both need to be the same

Reply:
Isn't both ascendening by default?

Reply:
anyway, i would explicitly specify it as ascending